ui/src/pages/MyAuctionsPage/MyAuctionsPage.tsx

```diff
@@ -1,7 +1,73 @@
-import React from 'react';
+import axios from 'axios';
+import { useState } from 'react';
+import { useQuery } from 'react-query';
+import AuctionList from '../../components/elements/AuctionList/AuctionList';
+import SearchInput from '../../components/elements/Search/Search';
+import { ToastContainer } from 'react-toastify';
+import useAuctionNextCursor from '../../hooks/useAuctionNextCursor';
+import getTokenFromCookies from '../../components/utils/getTokenFromCookies';
 
-function MyAuctionsPage() {
-  return <div>Hello</div>;
-}
+const getMyAuctions = async (cursor : any) => {
+  const token = getTokenFromCookies();
+  const url = `http://localhost:5167/api/user/auctions?pageSize=2&cursor=${cursor}`;
```

Comment:
Don't hardcode pagesize and cursor
store page size and cursor as url parameters on the frontend:
`frontend-url:3000?pageSize=2&cursor=cursorValue`. This is best practice for querying the data